lemarche/www/siaes/tests.py

```diff
@@ -540,6 +540,68 @@ def test_search_has_groups_should_filter(self):
         self.assertEqual(siaes[0].id, self.siae_without_group.id)
 
 
+class SiaeCAFilterTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.siae_without_ca_and_ca_api = SiaeFactory()
+        cls.siae_50k = SiaeFactory(ca=50000)
+        cls.siae_110k = SiaeFactory(ca=110000)
+        cls.siae_120k = SiaeFactory(api_entreprise_ca=120000)
+        cls.siae_550k = SiaeFactory(ca=550000, api_entreprise_ca=80000)
+        cls.siae_2m = SiaeFactory(ca=2000000, api_entreprise_ca=0)
+        cls.siae_7m = SiaeFactory(api_entreprise_ca=7000000)
+        cls.siae_50m = SiaeFactory(ca=0, api_entreprise_ca=50000000)
+
+    def test_search_query_empty(self):
+        url = reverse("siae:search_results")
+        response = self.client.get(url)
+        siaes = list(response.context["siaes"])
+        self.assertEqual(len(siaes), 8)
+
+    def test_search_ca_less_100k_filter(self):
+        url = reverse("siae:search_results") + "?ca=-100000"
```

Comment:
```suggestion
        url = f"{reverse("siae:search_results")}?ca={FORM_CA_CHOICES_RANGE_1}"
```
j'aurais peut être pas mis la valeur en brut comme ça et j'aurais mis la variable pour que ça soit plus maintenable :)

Reply:
Ah ba oui en effet..